assets/js/lesson_19/paparazzi_func.js

```diff
@@ -0,0 +1,7 @@
+function collectUserInfo(name) {
+    return (role) => {
+        return {name: name, role: role}
+    }
+}
```

Comment:
not bad. But 
1. this entry `{name: name, role: role}` can be shortened as `{name, role}` https://www.geeksforgeeks.org/shorthand-syntax-for-object-property-value-in-es6/
2. if you use function notation - so you should use it everywhere in the code, otherwise use arrow notation everywhere (just code style advice)
3. Try to rewrite this function using arrow function in one string